_posts/2020-02-09-week-1.md

```diff
@@ -31,6 +31,15 @@ View our detailed lesson plan [here](https://drive.google.com/open?id=10dzuF52k5
 - **Introduction to Scratch** (25 min): Watch Scratch tutotial, make accounts, work on scavenger hunt worksheet
 - **Wrap up** (5 min): Recap, demo the final maze activity
 
+### Video Materials
+We've produced exciting videos that go over the topics outlined in our lesson plan. These can serve as inspiration for how you could conduct your lessons, be used alongside your own explanations to help students visualize certain concepts, or even act as their own stand-alone lesson.
```

Comment:
As Lucas and I agreed, change intro to: We've produced exciting videos that go over the topics outlined in our lesson plan. These can serve as inspiration for how you could conduct your lessons. If you or the students don’t have time to watch the videos...check out our quick cheat sheet: https://drive.google.com/file/d/1ndrnZxA1OVSXp6b4z0ZJV3ePsPDaQmXC/view?usp=sharing